src/packages/files/files.ts

```diff
@@ -89,6 +90,58 @@ export default (app: Express) => {
     }
   });
 
+  /**
+   * @swagger
+   * /file/{id}:
+   *   get:
+   *     parameters:
+   *       - description: ID of the file
+   *         in: path
+   *         name: id
+   *         required: true
+   *         schema:
+   *           type: number
+   *     produces:
+   *       - application/json
+   *     responses:
+   *       200:
+   *         description: Fetches the file by id
+   *     summary: Get the file with the associated ID.
+   *     tags:
+   *       - Files
+   */
+  app.get('/file/:id', filesLimiter, auth, async (req, res) => {
+    const { params } = req;
+    const { id } = params;
```

Comment:
Is it possible for `id` to be `null`?

Reply:
It's not, if you call `/file/` without providing an `id` express compains.